index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+ <head>
+  <meta charset="utf-8">
+  <link href="style.css" type="text/css" rel="stylesheet">
+  <title>Curriculum Vitae</title>
+ </head>
+ <body>
+
+   <h1>Curriculum Vitae</h1>
+
+   <ol>
+    <li>Nikita Kashelevich</li>
```

Comment:
It was recommended to use semantic markup. Here I would like to see at least the use of semantic tags, such as header, footer, article, figure and other.